src/main/java/seedu/duke/userinterface/InputParser.java

```diff
@@ -169,32 +169,45 @@ public CliCommand getCommandFromInput(String userInput, AppState appState) throw
                 // TODO: implement adding pages
                 return new AddCommandNotebookMode(titleToAdd, contentToAdd, appState);
             }
+        case ListCommandTimetableMode.COMMAND_WORD:
+            if (appState.getAppMode() == AppMode.TIMETABLE) {
+                return new ListCommandTimetableMode(argument, appState);
+            } else {
+                return new ListCommandNotebookMode(argument, appState);
+            }
         case RemoveCommandTimetableMode.COMMAND_WORD:
             if (appState.getAppMode() == AppMode.TIMETABLE) {
                 return new RemoveCommandTimetableMode(parseTaskIndex(argument), appState);
             } else {
-                return new RemoveCommandNotebookMode(argument, appState);
+                String notebookTitleToRemove = "";
+                String sectionTitleToRemove = "";
+                int pageNumberToRemove = -1;
+                if (argument.contains("/n")) {
+                    notebookTitleToRemove = parseNotebookTitle(argument);
+                }
+                if (argument.contains("/s")) {
+                    sectionTitleToRemove = parseSectionTitle(argument);
+                }
+                if (argument.contains("/p")) {
+                    pageNumberToRemove = parsePageNumber(argument);
```

Comment:
Trivial, but you may want to use NOTEBOOK_DELIMITER, SECTION_DELIMITER and PAGE_DELIMITER respectively for this.